packages/video_player/example/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="org.tizen.video_player_tizen_example" version="1.0.0" api-version="6.0" xmlns="http://tizen.org/ns/packages">
```

Comment:
Is this plattform version limitation still intentional? I tested on a TM1 device (4.0) and watch emulator (4.0) with [this patch](https://github.com/flutter-tizen/engine/pull/60) and both basically worked (except for error messages regarding OnFrameAvailable failure and some flickering effect on emulator).

Reply:
Maybe you changed the value because of this issue https://github.com/flutter-tizen/flutter-tizen/issues/50 but you can now change it back to the value you originally intended.

Reply:
I updated the flutter-tizen and retest the example app, it's failed to install video player example in watch device(4.0) and emulator (4.0) with this patch.
```
path is /home/owner/share/tmp/sdk_tools/org.tizen.video_player_tizen_example-1.0.0.tpk
__return_cb req_id[1] pkg_type[tpk] pkgid[org.tizen.video_player_tizen_example] key[start] val[update]
__return_cb req_id[1] pkg_type[tpk] pkgid[org.tizen.video_player_tizen_example] key[install_percent] val[8]
__return_cb req_id[1] pkg_type[tpk] pkgid[org.tizen.video_player_tizen_example] key[install_percent] val[10]
__return_cb req_id[1] pkg_type[tpk] pkgid[org.tizen.video_player_tizen_example] key[install_percent] val[13]
__return_cb req_id[1] pkg_type[tpk] pkgid[org.tizen.video_player_tizen_example] key[install_percent] val[16]
__return_cb req_id[1] pkg_type[tpk] pkgid[org.tizen.video_player_tizen_example] key[error] val[-4]
__return_cb req_id[1] pkg_type[tpk] pkgid[org.tizen.video_player_tizen_example] key[end] val[fail]
processing result : Operation not allowed [-4] failed
```
And I change the version and target framework to 4.0, it stiil can't be launched.

Reply:
Haven't you said you already tried the same thing [before](https://github.com/flutter-tizen/plugins/pull/48#discussion_r602067209)? I provided a patch which fixes the error you mentioned in your comment so there shouldn't be any problem now, right?

Please make sure to apply these all:

```patch
$ git stash show -p
diff --git a/packages/video_player/example/tizen/Runner.csproj b/packages/video_player/example/tizen/Runner.csproj
index 8ebc2ab..497f928 100644
--- a/packages/video_player/example/tizen/Runner.csproj
+++ b/packages/video_player/example/tizen/Runner.csproj
@@ -2,7 +2,7 @@

   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>tizen60</TargetFramework>
+    <TargetFramework>tizen40</TargetFramework>
   </PropertyGroup>

   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
diff --git a/packages/video_player/example/tizen/tizen-manifest.xml b/packages/video_player/example/tizen/tizen-manifest.xml
index 0f8c6cd..3e5e375 100644
--- a/packages/video_player/example/tizen/tizen-manifest.xml
+++ b/packages/video_player/example/tizen/tizen-manifest.xml
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="org.tizen.video_player_tizen_example" version="1.0.0" api-version="6.0" xmlns="http://tizen.org/ns/packages">
+<manifest package="org.tizen.video_player_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
     <profile name="common"/>
-    <ui-application appid="org.tizen.video_player_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" taskmanage="true" nodisplay="false" api-version="6" launch_mode="single">
+    <ui-application appid="org.tizen.video_player_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" taskmanage="true" nodisplay="false" api-version="4" launch_mode="single">
         <label>video_player_tizen_example</label>
         <icon>ic_launcher.png</icon>
         <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true"/>
+        <metadata key="http://tizen.org/metadata/direct-launch" value="yes"/>
     </ui-application>
     <feature name="http://tizen.org/feature/screen.size.all"/>
     <privileges>
diff --git a/packages/video_player/tizen/project_def.prop b/packages/video_player/tizen/project_def.prop
index 6c93378..41d7cf6 100644
--- a/packages/video_player/tizen/project_def.prop
+++ b/packages/video_player/tizen/project_def.prop
@@ -3,7 +3,7 @@

 APPNAME = video_player_tizen_plugin
 type = sharedLib
-profile = common-5.5
+profile = common-4.0

 # Source files
```

Reply:
It can work on Tizen 4.0 now.